tox.ini

```diff
@@ -14,6 +15,8 @@ DJANGO =
     2.0: django20
     2.1: django21
     master: djangomaster
+OPTIONALS =
+    1: optionals
```

Comment:
Would prefer to override the `[testenv]` block instead of introducing an environment variable. eg, 
```ini
[testenv:py36-django20-optionals
deps = 
    {[testenv]deps}
    -rrequirments/requirments-optionals.txt
```

Reply:
How to run it on Travis then?  via an explicit TOXENV?

Reply:
Done it like that now (see https://github.com/encode/django-rest-framework/pull/6041).